cmd/expiration-mailer/main.go

```diff
@@ -205,15 +205,21 @@ func (m *mailer) processCerts(allCerts []core.Certificate) {
 			parsedCerts = append(parsedCerts, parsedCert)
 		}
 
+		if len(parsedCerts) == 0 {
```

Comment:
it's right here: https://github.com/letsencrypt/boulder/pull/1770/files#diff-0de749e99ab989dfcb38fab5231fe6a5R384